chessclient.py

```diff
@@ -2,21 +2,41 @@
 
 ENDIANNESS = "big"
 
+
+
```

Comment:
Try keep white-space changes out of commits. It distracts from the purpose of the commit.